test-utils/src/main/kotlin/Utils.kt

```diff
@@ -54,14 +54,14 @@ fun patchExpectedResult(
         if (newValue != null) replace(oldValue, newValue) else this
 
     return result.readText()
-        .replaceIfNotNull(custom)
         .replaceIfNotNull("<REPLACE_JAVA>", System.getProperty("java.version"))
         .replaceIfNotNull("<REPLACE_OS>", System.getProperty("os.name"))
         .replaceIfNotNull("<REPLACE_DEFINITION_FILE_PATH>", definitionFilePath)
         .replaceIfNotNull("<REPLACE_URL>", url)
         .replaceIfNotNull("<REPLACE_REVISION>", revision)
         .replaceIfNotNull("<REPLACE_PATH>", path)
         .replaceIfNotNull("<REPLACE_URL_PROCESSED>", urlProcessed)
+        .let { custom.fold(it) { text, strings -> text.replaceIfNotNull(strings) } }
```

Comment:
Could you please keep this as the first replacement? The idea behind doing this first was that a custom replacement should also be able to override any of the hard-coded replacements.

Reply:
Done.